setup.cfg

```
@@ -1,14 +1,14 @@
 [metadata]
 name = pythoncalculator
 version = 0.0.1
-author = Example Author
-author_email = author@example.com
+author = Twin Karmakharm
+author_email = t.karmakharm@sheffield.ac.uk
 description = Perform basic calculations in python
 long_description = file: README.md
 long_description_content_type = text/markdown
-url = https://github.com/<owner-account>/python-calculator
+url = https://github.com/tkarma/python-calculator
```

I think the URL here might be wrong, should it perhaps be....
```suggestion
url = https://github.com/twinkarma/python-calculator
```

Also applies to the below URL.